proposals-CG-WG.md

```diff
@@ -1,82 +1,78 @@
-# FedID CG/WG Process Mechanics
+# CG/WG Proposal Stages
+
+Author: @samuelgoto
+Created: 08/11/2024
+Last update: 09/11/2024
+Status: under review
 
 This is a proposal to break proposals into 5 stages of maturity, with clear guidelines and requirements to advance them:
 
-* [Stage 0](#stage0): Understand the Problem Space
-* [Stage 1](#stage1): Incubate Proposals
-* [Stage 2](#stage2): Specify a Preferred Proposal
-* [Stage 3](#stage3): Increase Deployment Confidence
-* [Stage 4](#stage4): Publish a Proposed Recommendation
+* [Stage 0](#stage0): **Exploration** of the Problem Space
+* [Stage 1](#stage1): **Incubation** of competing Alternatives
+* [Stage 2](#stage2): **Formalization** a preferred Proposal
+* [Stage 3](#stage3): **Independent Implementations** of the preferred Proposal
+* [Stage 4](#stage4): **Publication** of a Proposed Recommendation
+
+# Stage 0: Exploration
+
+The purpose of Stage 0 proposals is to allow anyone to raise and explore the Problem Space without asking for permission. 
+
+  * What's needed from individuals?
```

Comment:
Ah, that occurred to me too, but I figured it would be constructive to separate what the champions need to bring and what the reviewers need to agree to. That seemed like an important distinction, wdyt?

Reply:
done